src/components/ReactSlider/ReactSlider.md

```diff
@@ -9,6 +9,37 @@ Single slider, similar to `<input type="range" defaultValue={0} />`
 />
 ```
 
+Slider with marks
+
+```jsx
+<ReactSlider
+    className="horizontal-slider"
+    marks={[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}
+    renderMark={(props) => <span className="example-dot" {...props} />}
+    min={0}
+    max={9}
+    thumbClassName="example-thumb"
+    trackClassName="example-track"
+    renderThumb={(props, state) => <div {...props}>{state.valueNow}</div>}
+/>
+```
+
+Slider with marks and right to left
```

Comment:
This example feels a little edge-casey -- I think it's good to have an example for `inverse`, but the marks feel out of place. I'm guessing this was to make sure marks were also inversed?

Reply:
This is the reason, to show the marks in the right place once inverted, I can change theme if you prefer